# vrep start/stop
int32 vrep_state
int32 SIM_STOPPED = 0
int32 SIM_RUNNING = 1

# vrep action state
int32 action_state
int32 READY_FOR_ACTION = 0
int32 ACTION_RUNNING = 1